### ISSUE TITLE
aot is broken again

### ISSUE THREAD

**Issue by seiyria** [farin99/angular2-virtual-select#14]
```
Error: Error at C:/Users/u34414/Desktop/webapp/compiled/node_modules/angular2-vi
rtual-select/dist/virtual-scroll.ngfactory.ts:210:18: Property 'contentElementRe
f' is protected and only accessible within class 'VirtualScrollComponent' and it
s subclasses.
Error at C:/Users/u34414/Desktop/webapp/compiled/node_modules/angular2-virtual-s
elect/dist/virtual-scroll.ngfactory.ts:224:41: Property 'scrollHeight' is privat
e and only accessible within class 'VirtualScrollComponent'.
Error at C:/Users/u34414/Desktop/webapp/compiled/node_modules/angular2-virtual-s
elect/dist/virtual-scroll.ngfactory.ts:229:58: Property 'topPadding' is private
and only accessible within class 'VirtualScrollComponent'.
```

Seems like some accessor fixes, which shouldn't be too bad!

**Comment by farin99** [comment]
Fixed in latest npm (1.1.15).

**Comment by seiyria** [comment]
Thanks!
